src/signals/incident-management/containers/IncidentOverviewPage/components/Th/Th.tsx

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import type { SortOptions } from '../contants'
+import SortIcon from '../SortIcon'
+
+const ThComponent = ({
+  ordering,
+  changeOrder,
+  sortOption,
+  headerText,
+  StyledComponent,
+}: {
+  ordering?: SortOptions
+  changeOrder: (column: SortOptions) => void
+  sortOption: SortOptions
+  headerText: string
+  StyledComponent: any
+}) => {
+  return (
+    <StyledComponent
+      $underline={
+        ordering && ordering.replace('-', '') === sortOption.replace('-', '')
```

Comment:
This funtionality also exists in Sort Icon. Maybe good to have a property or function that identifies the active sort?